java/spacewalk-java.spec

```diff
@@ -116,8 +111,14 @@ BuildRequires:  %{apache_commons_discovery}
 BuildRequires:  %{apache_commons_el}
 BuildRequires:  %{apache_commons_fileupload}
 BuildRequires:  %{apache_commons_validator}
+%if 0%{?rhel} >= 8 || 0%{?fedora}
+BuildRequires:  java-11-openjdk-devel
+BuildRequires:	maven-javadoc-plugin
+%else
 BuildRequires:  java-devel >= %{java_version}
+%endif
 BuildRequires:  java-saml
+BuildRequires:  javamail
```

Comment:
In line 96 we have the same inside of a "%if suse". When this is also needed on CentOS, please remove it from above.

Reply:
Thx!